internal/proto/controller/api/resources/sessionrecordings/v1/session_recording.proto

```diff
@@ -130,10 +130,10 @@ message HostCatalog {
 // Host describes the Host that was chosen for the recorded session.
 message Host {
   // The ID of the Host
-  string id = 1; // @gotags: class:"public"
+  string id = 1; // @gotags: class:"public" eventstream:"observation"
 
   // The Host Catalog this Host is in
-  HostCatalog host_catalog = 2 [json_name = "host_catalog"]; // @gotags: class:"public"
+  HostCatalog host_catalog = 2 [json_name = "host_catalog"]; // @gotags: class:"public" eventstream:"observation"
```

Comment:
I don't think this will have any effect (`HostCatalog` is a message type). Both the `class` and `eventstream` tags can be removed, I think.